src/hrflow_connectors/connectors/poleemploi/connector.py

```diff
@@ -0,0 +1,108 @@
+import typing as t
+
+from hrflow_connectors.connectors.hrflow.warehouse import HrFlowJobWarehouse
+from hrflow_connectors.connectors.poleemploi.warehouse import PoleEmploiJobWarehouse
+from hrflow_connectors.core import BaseActionParameters, Connector, ConnectorAction
+
+
+def get_job_location(poleemploi_location: t.Union[t.Dict, None]) -> t.Dict:
+    if poleemploi_location is None:
+        return dict(lat=None, lng=None, text="")
+
+    lat = poleemploi_location.get("latitude")
+    lat = float(lat) if lat is not None else lat
+
+    lng = poleemploi_location.get("longitude")
+    lng = float(lng) if lng is not None else lng
+
+    concatenate = []
+    for field in ["libelle", "codePostal"]:
+        if poleemploi_location.get(field):
+            concatenate.append(poleemploi_location.get(field))
```

Comment:
return " ".join([poleemploi_location.get(field) for field in ["libelle", "codePostal"] if poleemploi_location.get(field)])

Always try to use one-liners and compact functions as much as possible 

when you first code code with for loop and test and then refactor your code